components/generic_modal.scss

```diff
@@ -14,6 +14,21 @@
             box-sizing: border-box;
             border-radius: 4px;
             height: 40px;
+
+            &.has-error {
```

Comment:
Can you move these CSS styles out of the `.GenericModal` class? We're going to want to be able to use this component elsewhere, so it should be self-contained. At the same time, can you rename the max-length class to `MaxLengthInput` to match the component? That's a naming convention we're trying to start using for CSS at the root of a component

Reply:
@hmhealey I had 2 concerns and didn't take that route,

1. inputs may vary on height, or padding, etc, so styling the validation message outside the input it's used may also 
break re-usability (it's an absolute positioned element), so my thought was to just pass classes like `.has-error` which other component may override so it would be easier re-usable.

2. `QuickInput` is being styled inside `.GenericModal`, so it seemed like the reasonable place for `MaxLengthInput` to be styled as well.

Please let me know your thoughts on 1.

Reply:
>1. inputs may vary on height, or padding, etc, so styling the validation message outside the input it's used may also
>break re-usability (it's an absolute positioned element), so my thought was to just pass classes like `.has-error` which other component may override so it would be easier re-usable.

That's fair. I think we'll eventually end up making those inputs more consistent since there's a big design system overhaul coming eventually, but we can do that once that's ready.

> 2. `QuickInput` is being styled inside `.GenericModal`, so it seemed like the reasonable place for `MaxLengthInput` to be styled as well.

I didn't realize we were also doing that in there. I guess trying to move `MaxLengthInput` out of there but not `QuickInput` would probably mean we'd run into issues with CSS specificity causing styles to override each other. That's another reason to wait for that until we do the bigger overhaul.

I'd appreciate the renaming still though